### ISSUE TITLE
Include the app in the Application Center

### ISSUE THREAD

**Issue by megatux** [donadigo/wingpanel-indicator-namarupa#2]
Is this possible?

**Comment by sprite-1** [comment]
From the [elementary OS publishing guidelines](https://github.com/elementary/houston/wiki/Before-You-Publish) (emphasis mine)

> AppCenter supports publishing apps that meet these requirements; **extensions or plugins (eg. to the Panel, System Settings, or other apps) are not currently supported**.

**Comment by megatux** [comment]
oh, right. I forgot about that! Do you want to keep this issue opened as a reminder of this pending feature? We can link to main the App. Center issue.

**Comment by sprite-1** [comment]
You'll have to ask @donadigo

**Comment by donadigo** [comment]
@sprite-1 yes we can as well keep it here. You're right that at this point AppCenter doesn't support plugins and I don't know when it will. I guess that for now we can just provide .deb's. That doesn't mean I wouldn't like to put it on AppCenter for streamlined install, if it was a possibility, I would totally do that.
